lte/gateway/configs/pipelined.yml

```diff
@@ -194,7 +194,7 @@ ovs_internal_conntrack_port_number: 15579
 ovs_internal_conntrack_fwd_tbl_number: 202
 
 # For 5G Functionality Support flag
```

Comment:
It might be helpful to add a line about which config from the Orc8r this would override

Reply:
+1, also please specify that this is commented out to give the orc8r streamed config precedence above the local config

Reply:
Added the comments